<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateUser</class>
 <widget class="QDialog" name="updateUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mettre a jour</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="5" column="0" colspan="2">
      <widget class="QLineEdit" name="adresseEMailLineEdit"/>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="adresseEMailLabel">
       <property name="text">
        <string>Adresse e-mail</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLineEdit" name="prNomLineEdit"/>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="prNomLabel">
       <property name="text">
        <string>Pr√©nom</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="nomLabel">
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QLineEdit" name="nomLineEdit"/>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QLineEdit" name="motDePasseLineEdit"/>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QLabel" name="motDePasseLabel">
       <property name="text">
        <string>Mot de passe</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Mettre a jour</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font-size: 20px;
color: &quot;#FCA311&quot;;
margin-bottom: 16px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLineEdit" name="idLineEdit"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="idLabel">
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
